### ISSUE TITLE
Minor error if we try to output .c file without outputting shellcode

### ISSUE THREAD

**Issue by paraschetal** [zscproject/OWASP-ZSC#89]
```
Output assembly code?(y or n)> n
Output shellcode to screen?(y or n)> n
Shellcode output to a .c file?(y or n)> y
Target .c file?> ../testZSC.c
Traceback (most recent call last):
  File "/usr/share/owasp_zsc/zsc.py", line 23, in <module>
    main()  #execute main function
  File "/usr/share/owasp_zsc/zsc.py", line 18, in main
    _interface()
  File "/usr/share/owasp_zsc/core/controller.py", line 23, in _interface
    run.engine(commands)  #run engine with user friendly interface
  File "/usr/share/owasp_zsc/core/run.py", line 261, in engine
    getcommand(commands)
  File "/usr/share/owasp_zsc/core/run.py", line 197, in getcommand
    file_output(target, func, data, os, encode, shellcode, shellcode_op)
UnboundLocalError: local variable 'shellcode_op' referenced before assignment
```

Sending a PR in a moment.

**Comment by paraschetal** [comment]
@Ali-Razmjoo 
PR sent

**Comment by Ali-Razmjoo** [comment]
Merged #90 thanks.
